.ci/build_packages/tests.sh

```diff
@@ -31,11 +68,10 @@ emqx_prepare(){
 
 emqx_test(){
     cd "${PACKAGE_PATH}"
+    local packagename="${PACKAGE_FILE_NAME}"
 
-    for var in "$PACKAGE_PATH"/"${EMQX_NAME}"-*;do
-        case ${var##*.} in
+        case ${packagename##*.} in
```

Comment:
Now that the package suffix is fully determined above, maybe we could directly case split on `$PKG_SUFFIX`?